src/notifications/changing_strings.md

```diff
@@ -0,0 +1,31 @@
+# Changing default strings
```

Comment:
⚠️ Exemplary suggestion.

If previous suggestion to use title cases on titles is to be implemented, it should affect all other links and the titles on the pages themselves.
```suggestion
# Changing Default Strings
```